cypress/e2e/Telechargement/TableauBiospecimens.cy.ts

```diff
@@ -21,7 +21,7 @@ describe('Page Data Exploration (Biospecimens) - Exporter les biospecimens en TS
     cy.validateFileHeaders('ExportTableauBiospecimens.json');
   });
 
-  it('Valider le contenu du fichier', () => {
+  it('Valider le contenu du fichier [CQDG-500]', () => {
```

Comment:
Je ne suis pas sur mais le ticket 500 est en backlog
```suggestion
  it('Valider le contenu du fichier [CQDG-513]', () => {
```

Reply:
vu avec karine c'est good et voulu